app/models/invalid_identities.rb

```diff
@@ -0,0 +1,20 @@
+class InvalidIdentities
+
```

Comment:
Can you explain creating a model (and then also the matching spec file) for this? Are we likely to use this again? Seems like way over-engineering to me, but I could totally be missing something here.

Also, it seems like if we ARE going to use a model to define an "invalid identity" then the definition should be in said model. So if lacking an email makes an identity invalid, that should be in the model. As it stands, an "invalid identity" is basically just whatever identity gets passed into the model.

I personally don't think a model is needed, but if we DO want a model, we should go all the way and have it a self-contained data-type definition.